kitchen/orb.yml

```diff
@@ -128,6 +128,35 @@ jobs:
           path: .kitchen/logs
           when: always
 
+  dokken-single-ipv6:
+    executor: chef
+    parameters:
+      suite:
+        description: Test Kitchen suite name
+        type: string
+      channel:
+        description: ChefDK channel stable or current
+        type: string
+        default: current
+    environment:
+      KITCHEN_LOCAL_YAML: kitchen.dokken.yml
+      CHEF_LICENSE: "accept-no-persist"
+    steps:
```

Comment:
I wonder if we could have this as an additional/optional block in the normal `dokken-single`?